src/scene/mesh.js

```diff
@@ -32,6 +32,7 @@ Object.assign(pc, function () {
      * {@link pc.GraphicsDevice#draw}. The primitive is ordered based on render style like the indexBuffer property.
      * @property {pc.BoundingBox} aabb The axis-aligned bounding box for the object space vertices of this mesh.
      * @property {pc.Skin} [skin] The skin data (if any) that drives skinned mesh animations for this mesh.
+     * @property {pc.Morph} [morph] The list of pc.MorphTarget (if any) for this mesh.

```

Comment:
`pc.MorphTarget` -> `pc.Morph` in the property description?

Reply:
I was trying to describe the `pc.Morph` (which is basically a list of `pc.MorphTarget`). Perhaps it's better to just say `The pc.Morph object (if any) for this mesh.`